snooty/types.py

```diff
@@ -119,6 +126,13 @@ class ProjectConfig:
     default_domain: Optional[str] = field(default=None)
     title: str = field(default="untitled")
     source: str = field(default="source")
+    banner: Dict[str, str] = field(default_factory=dict)
+    # banner_node contains an instantiated banner node with target data
+    banner_node: BannerConfig = field(
+        default_factory=lambda: BannerConfig(
+            "", n.Directive((-1,), [], "", "banner", [], {})
```

Comment:
is using `-1` for the `Line` tuple valid or sensible as a default value?

In general I couldn't find an existing example of instantiating a Directive with a line value when that Directive might be placed arbitrarily in the AST.